config/initializers/timezone.rb

```diff
@@ -0,0 +1,2 @@
+# Set the timezone for the JavascriptAgent (libv8 only relies on the TZ variable)
+ENV['TZ'] = Time.zone.tzinfo.canonical_identifier
```

Comment:
My thought was to use the `TIMEZONE` .env variable as truth for the timezone the user selects. If we would callback to the system wide `TZ` when it's present, the only way to have a consistent timezone setting would be to add `TZ` to `.env` as well.